### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by jedbrown**
I would like to get explicit approval on this PR from all CEED developers as an affirmative statement of intent to abide by and enforce the code of conduct.

Adapted from Contributor Covenant. Enforcement section lists group contact (ceed-users@llnl.gov) and confidential contacts.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported publicly at ceed-users@llnl.gov or confidentially by contacting jed@jedbrown.org or valeria.barra@colorado.edu. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (tzanio) [inline CODE_OF_CONDUCT.md]: I'd like to add tzanio@llnl.gov to the list of emails.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# libCEED Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (tzanio) [inline CODE_OF_CONDUCT.md]: Couldn't agree more

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported publicly at ceed-users@llnl.gov or confidentially by contacting jed@jedbrown.org, valeria.barra@colorado.edu, or tzanio@llnl.gov. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (jedbrown) [inline CODE_OF_CONDUCT.md]: Revised @tzanio.

**Comment by stomov** [comment]
This sounds very good.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (stomov) [reply CODE_OF_CONDUCT.md]: This sounds very good.

**Pull request merged**